### ISSUE TITLE
Space key does not work when using Full Keyboard Access

### ISSUE THREAD

**Issue by zrhoffman** [ish-app/ish#2657]
Version: 1.3.2
Device: iPhone 17 Pro running iOS 26.1

Steps to reproduce:
- Connect a keyboard
- Go to Accessibility -> Keyboards & Typing -> Full Keyboard Access and enable Full Keyboard Access
- Open iSH
- Type `echo my message` and press Enter

Expected result:
```
my message
```

Actual result:

```
ash: echomymessage: not found
```

Screenshot:
<img width="1206" height="2622" alt="Screenshot of reproduction of Issue #2648" src="https://github.com/user-attachments/assets/be8fc15d-ed89-4737-86e6-084d357cb3dc" />

**Comment by Nayelianne** [comment]
Can confirm this happening on ipadOS 18 on the latest app store version of the app as well.
A temporary solution that does not involve turning off Full Keyboard Access is using the shortcut "Control + Option + Command(⌘) + P". You should then be able to type normally.
But if you let your ipad fall asleep after that, you need to repeat the shortcut twice to reclaim regular typing functionality.
